src/molecules/related-story-card/related-story-card.tsx

```diff
@@ -32,10 +32,16 @@ export const RelatedStoryCard = ({ story, config }: RelatedStoryCardTypes) => {
 
   const languageCode = get(config, ["publisherConfig", "language", "ietf-code"]);
   const localizationOpts = get(config, ["opts", "featureConfig", "localization"], {});
+  let timeStamp = lastPublishedAt;
+  const storyCardTime = get(config, ["opts", "featureConfig", "storyCardTime"], null);
+  const customTimeStamp = storyCardTime && typeof storyCardTime === "function" && storyCardTime(config, story);
+  if (customTimeStamp) {
+    timeStamp = customTimeStamp;
+  }
```

Comment:
```suggestion
  if (storyCardTime && typeof storyCardTime === "function") {
    timeStamp = storyCardTime(config, story);
  }
```

Reply:
@ags1773 We don't know if storyCardTime(config, story) is truthy in this version

Reply:
It doesn't matter.. we can assume that the function returns a proper value

Reply:
+1